Dockerfile

```diff
@@ -109,56 +109,15 @@ RUN chmod +x /usr/local/bin/docker-entrypoint
 ENTRYPOINT ["docker-entrypoint"]
 CMD ["php-fpm"]
 
+FROM caddy:${CADDY_VERSION}-builder-alpine AS symfony_caddy_builder
 
-# "nginx" stage
-# depends on the "php" stage above
-FROM nginx:${NGINX_VERSION}-alpine AS symfony_nginx
+RUN xcaddy build \
+    --with github.com/dunglas/vulcain/caddy
```

Comment:
github.com/dunglas/vulcain/tree/master/caddy ? Missing `tree/master`

Reply:
No it's the name of the Go module, it's intended.